src/engine/controls/cuecontrol.cpp

```diff
@@ -1769,7 +1766,7 @@ HotcueControl::HotcueControl(QString group, int i)
 
     // The rgba value  of the color assigned to this color.
     m_hotcueColor = new ControlObject(keyForControl(i, "color"));
-    m_hotcueColor->set(kNoColorControlValue);
+    m_hotcueColor->set(QRgb(ColorPalette::kDefaultCueColor));
```

Comment:
> what are "All" other COs? Only the position CO is set to -1 because it represents the invalid value, which has a programmatic use case.

You're right. I was under the impression that more CO's were using `-1` but it looks like that's not the case.

> Compared to this, initializing the color CO with the default color is a good choice.

Ok, I'd prefer `0` though which is also a valid color.
![2020-03-20-184406_975x1274_scrot](https://user-images.githubusercontent.com/1834516/77191217-e4aa4400-6ada-11ea-8940-5217fe9c771e.png)